sleigh/sleigh-compile/src/constructor/disasm_actions.rs

```diff
@@ -91,7 +81,10 @@ impl ResolveIdent for DisasmConstantValue {
             Some(Local::Field(id)) => Ok(Self::LocalField(id)),
             Some(Local::InstStart) => Ok(Self::InstStart),
             Some(Local::InstNext) => Ok(Self::InstNext),
-            Some(other) => Err(format!("{:?}<{}> in disasm expr", other, scope.debug(&ident))),
+            Some(other) => {
+                tracing::error!("{:?}<{}> in disasm expr", other, scope.debug(&ident));
```

Comment:
Are these logs necessary? They just have the exact same info returned by the error message, and can be logged at a higher level.